Gems/PhysX/Code/Source/EditorColliderComponent.h

```diff
@@ -215,22 +205,20 @@ namespace PhysX
         AZ::Transform GetManipulatorSpace() const override;
         AZ::Quaternion GetRotationOffset() const override;
 
-        // AZ::Render::MeshComponentNotificationBus
-        void OnModelReady(const AZ::Data::Asset<AZ::RPI::ModelAsset>& modelAsset,
-            const AZ::Data::Instance<AZ::RPI::Model>& model) override;
-
-        // PhysX::ColliderShapeBus
+        // PhysX::ColliderShapeBus overrides ...
         AZ::Aabb GetColliderShapeAabb() override;
         bool IsTrigger() override;
 
-        // PhysX::EditorColliderComponentRequestBus
+        // PhysX::EditorColliderComponentRequestBus overrides ...
         void SetColliderOffset(const AZ::Vector3& offset) override;
```

Comment:
Does this stuff end up getting duplicated between the two collider components? How much stuff is duplicated?

Reply:
The `EditorColliderComponentRequestBus` implementation is very similar in both and the skeleton of both classes look the same, but eventually they arrive to do different things in private functions. As a first pass of splitting the component in 2 I think it makes sense as it is now, but in the future we can take another look and see if it's worth refactoring something out. 

I'm glad we're doing this split because I noticed `EditorMeshColliderComponent` has to do a lot than `EditorColliderComponent` (primitive one) just for having to deal with an asset and custom error checking, and it's great all that is out of the primitive collider.

Reply:
Cool, sounds good.